products/nodejs.md

```diff
@@ -17,6 +17,13 @@ command: node --version
 releaseDateColumn: true
 sortReleasesBy: 'releaseCycle'
 releases:
+  - releaseCycle: "18"
+    release: 2022-04-19
+    <!--- Change to true after 2022-10-25 --->
```

Comment:
`# Change to true after 2022-10-25 `
instead of
`<!--- Change to true after 2022-10-25 --->`

Reply:
Thanks. I don't why vscode suggested this format.

Reply:
Thanks. I don't know why vscode suggested this format.